Docker/itksoftwareguide-bin/README.md

```diff
@@ -1 +1,21 @@
+Description
+-----------
+
 Holds the volume with the build for the Software Guide.
+
+Volumes
+-------
+
+Source folder: /home/itk/src/ITKSoftwareGuide
+Binary folder: /home/itk/bin/ITKSoftwareGuide
+
+Usage
+-----
+
+Build in the docker container:
+  ./run.sh itksoftwareguide-bin
+
+Use ITKSoftwareGuide outside of the docker container:
+  ./run.sh itksoftwareguide-bin \
+    -v ${HostITKSoftwareGuidSourceDirectory}:/home/itk/src/ITKSoftwareGuide \
+    -v ${HostITKSoftwareGuidBinaryDirectory}:/home/itk/bin/ITKSoftwareGuide
```

Comment:
:+1:  May be https://cmake.org/download/ could be improved to include this ? And also the minimum required GLIBC ...